Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Issues.hs

```diff
@@ -67,3 +68,10 @@ findAllIssue merchantId mbLimit mbOffset fromDate toDate = Esq.findAll $ do
   where
     limitVal = min (maybe 10 fromIntegral mbLimit) 10
     offsetVal = maybe 0 fromIntegral mbOffset
+
+findByRideId :: Transactionable m => Maybe (Id R.Ride) -> m (Maybe Text)
```

Comment:
this query make no sense when `rideId = Nothing`. Can you change type:

`findByRideId :: Transactionable m => Id R.Ride -> m (Maybe (Id Issue))`
or
`findByRideId :: Transactionable m => Id R.Ride -> m (Maybe Issue)`